### ISSUE TITLE
Player Controls: Images must have alternate text

### ISSUE THREAD

**Issue by davefollett** [codingblocks/podcast-app#229]
## 💥 Impact rating
critical

## 👣 Steps to reproduce
Add a podcast to the queue, start playing it, pause it, and then [run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test).

## ⚛️ Element(s) with issue
1. Skip 10 seconds backwards player control button
1. Play / Pause player control button
1. Skip 30 seconds forward player control button

## 🛠️ Recommended fix
Fix at least one (1) of these issues:
* Element does not have an alt attribute
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
* Element has no title attribute or the title attribute is empty
* Element's default semantics were not overridden with role="presentation"
* Element's default semantics were not overridden with role="none"

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/image-alt?application=AxeChrome
